### PR TITLE
Add lb-nightly packages

### PR DESCRIPTION
**Pull request by maszyman**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/lb-nightly-functions/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+test:
+  imports:
+    - lb
+    - lb.nightly.functions
+  commands:
+    - pip check
+    - lb-wrapcmd --help
+  requires:
+    - git
```
Comment (chrisburr) [inline recipes/lb-nightly-functions/meta.yaml:46]: Should this be a full runtime dependency?

Comment (maszyman) [reply recipes/lb-nightly-functions/meta.yaml:46]: I'm not sure, but it fixed [this](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=409235&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=3417) (inspired by https://github.com/conda-forge/staged-recipes/pull/10958#issuecomment-625625359).

Comment (chrisburr) [reply recipes/lb-nightly-functions/meta.yaml:46]: Ah yes, `gitpython` is intentionally packaged to allow it to use the a non-conda-forge `git` binary.

(When deploying this you should probably add `git` to the `environment.yaml` spec you use.)

File: recipes/lb-nightly-rpc/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lb-nightly-rpc-{{ version }}.tar.gz
+  sha256: 8c331c51c969b4fc7c18b85b8f4fc0c9b97cb56eea78963986cd14bc4cb93c7c
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
```
Comment (chrisburr) [inline recipes/lb-nightly-rpc/meta.yaml:16]: If the package is noarch it doesn't matter if the windows CI fails with missing dependencies:
```suggestion
  noarch: python
```

Comment (maszyman) [reply recipes/lb-nightly-rpc/meta.yaml]: Ok,  thanks ;-)

**Pull request merged**